docs/root/api-v3/common_messages/common_messages.rst

```diff
@@ -24,6 +24,8 @@ Common messages
   ../extensions/common/ratelimit/v3/ratelimit.proto
   ../extensions/filters/common/fault/v3/fault.proto
   ../extensions/network/socket_interface/v3/default_socket_interface.proto
+  ../extensions/network/dns_resolver/cares/v3/cares_dns_resolver.proto
+  ../extensions/network/dns_resolver/apple/v3/apple_dns_resolver.proto
```

Comment:
This should be moved into a new section about DNS resolvers, close to the other relevant network protos. It's not a common message.

Reply:
@mattklein123 , thanks for reviewing. 

Yes, moved them docs/root/api-v3/config/dns_resolver/dns_resolver.rst.